bin/local-env/start.sh

```diff
@@ -9,12 +9,6 @@ set -e
 # Change to the expected directory
 cd "$(dirname "$0")/../.."
 
-# Check whether Node and NVM are installed
-. "$(dirname "$0")/install-node-nvm.sh"
-
-# Check whether Composer installed
-. "$(dirname "$0")/install-composer.sh"
```

Comment:
Instead of deleting these files, maybe better to add a flag to skip them or something.

Reply:
I don't think these files have ever proven useful.

Not having the latest versions of dependencies should not block you from spinning up docker environment.

And even if you have the latest versions it will still needlessly run `npm install`/`composer install` and just slow things down needlessly.

This is why I have decided to just remove them.